alternator-test/conftest.py

```diff
@@ -43,6 +43,8 @@
 def pytest_addoption(parser):
     parser.addoption("--aws", action="store_true",
         help="run against AWS instead of a local Scylla installation")
+    parser.addoption("--https", action="store_true",
+        help="communicate via HTTPS protocol instead of HTTP")
```

Comment:
Nitpick: this only effects the local url, not aws, so maybe the description should say something like "run against a local Scylla installation with the HTTPS protocol instead of HTTP."